lib/gooddata/lcm/synchronize_ldm_helper.rb

```diff
@@ -0,0 +1,93 @@
+module GoodData
+  module LCM2
+    class SynchronizeLdmHelper
+      class << self
+        def synchronize_ldm(from, segment_info, client, opts = {})
```

Comment:
Takes parameters ['client', 'from', 'opts', 'segment_info'] to 3 methods (DataClump)